src/network.rs

```diff
@@ -1,11 +1,11 @@
 //! Core deploy information
 
-use std::collections::{HashMap, HashSet};
-
-use crate::{
-    bridge::BridgeConfiguration,
-    common::{deser_nomad_number, NameOrDomain, NomadIdentifier, NomadLocator},
+use crate::bridge::BridgeConfiguration;
+use nomad_types::{
+    deser_nomad_number_u32, deser_nomad_number_u64, deser_nomad_number_u8, NameOrDomain,
```

Comment:
Implementations come from [this macro in rust PR](https://github.com/nomad-xyz/rust/pull/30/files#diff-877b65846e407bff372ed4e4f5e2f13c4aefe72f915c7404d7798b0387f849eeR1-R53)